pages/strategic-plan.md

```diff
@@ -1,141 +0,0 @@
----
-layout: page-fullwidth
-title: "Our Strategic Plan for 2020-2025"
```

Comment:
I'm fine with cleaning the file out but again it might be nice to leave some breadcrumbs (and a template) to help us structure such a page if we had one in the future

Reply:
Any of the existing files under `/pages/` will serve a future author as a fine template.

Reply:
In this case, I think you are right that if we need this someone will probably remember where it came from